File: 8393.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int givenNumber = sc.nextInt();
        int output = 0;
        sc.close();
        
        for(int i = 0; givenNumber >= i; i++)
            output = output + i;
        
        System.out.println(output);
    }
}

File: 8393.c
#include<stdio.h>

int main(){
    int givenNumber = 0, output = 0;
    scanf("%d", &givenNumber);
    
    for(int i = 1; givenNumber >= i; i++)
        output = output + i;
    
    printf("%d", output);
    
    return 0;
}
